pullreq_pipeline.go

```diff
@@ -68,3 +69,57 @@ func createPullRequestBranch(log *logrus.Entry, org, repo, pr, action string) er
 	log.Info("Pipeline is expected to start automatically")
 	return nil
 }
+
+func deleteStaleGitlabPRBranch(log *logrus.Entry, pr *github.PullRequestEvent, conf *config) error {
```

Comment:
For consistency, consider the same interface as the function above

```suggestion
func deleteStaleGitlabPRBranch(log *logrus.Entry, org, repo, pr, action string) error {
```

Reply:
I agreed, looking at it, but from looking in the main code, I actually think this is the saner interface, so leaving for now :)